services/src/handlers/wms.rs

```diff
@@ -361,13 +358,14 @@ mod tests {
     use actix_web::http::Method;
     use actix_web_httpauth::headers::authorization::Bearer;
     use geoengine_datatypes::operations::image::RgbaColor;
-    use geoengine_datatypes::primitives::SpatialPartition2D;
+    // use geoengine_datatypes::primitives::SpatialPartition2D;
```

Comment:
remove the commented out code

Reply:
we will need it if the test is fixed i guess